src/site-server.ts

```diff
@@ -98,7 +98,7 @@ export class SiteServer {
 			);
 		}
 
-		console.log( 'Starting server with options', sanitizeForLogging( options ) );
```

Comment:
Since we are removing `sanitizeForLogging` here, we can also remove its import (in both `src/site-server.ts` and `src/ipc-handlers.ts`).

Reply:
Good catch, removed in 13bfd784edae44f103bdaa3c879be5faf8218397

Reply:
Thanks. 🙂

Reply:
It is also in `src/ipc-handlers.ts`.

Reply:
Thanks: 98a0fa1aec9f4d3f31f16655601dbf93ff9d2b08. It's easy to miss now as we have 110 lint warnings. 🤷